### ISSUE TITLE
Clicking on edition, level or pages on the adventure details page should bring you back to the search page

### ISSUE THREAD

**Issue by cmfcmf** [AdventureLookup/AdventureLookup#320]
Clicking on the grey tags at the top here:
![image](https://user-images.githubusercontent.com/2145092/82141174-5c26f600-9834-11ea-96b2-d7d16cfb7423.png)
should bring you back to the search page, similar to how clicking on one of the authors brings you back to search.

Clicking on the number of pages should probably not search for that exact number, but rather for that approximate number of pages. For example, clicking on `36` pages should open up the search for adventures with `>= 36*0.85` and `<= 36*1.15` pages. Not sure about the exact percentage though (15% in the example).

**Comment by damienBAUDET** [comment]
hi, if it's possible i will try to work on it.

**Comment by cmfcmf** [comment]
Of course! :tada:

**Comment by damienBAUDET** [comment]
Hi, I have a question about the number of pages. How the macro search works with integer in /app/Resources/views/adventure/_details.html.twig ? I don't know how put the min and the max in. Thanks for your help 😄

**Comment by cmfcmf** [comment]
I think you should be able to change the search macro to accept min/max for integers like so (untested):
```diff
diff --git a/app/Resources/views/adventure/_details.html.twig b/app/Resources/views/adventure/_details.html.twig
index f6c8f3d..81ccb6e 100644
--- a/app/Resources/views/adventure/_details.html.twig
+++ b/app/Resources/views/adventure/_details.html.twig
@@ -2,8 +2,8 @@
     {% apply spaceless %}
         {% if isInteger %}
             {% set path = path('adventure_index', {
-                ('f[' ~ fieldName ~ '][v][min]'): value,
-                ('f[' ~ fieldName ~ '][v][max]'): value,
+                ('f[' ~ fieldName ~ '][v][min]'): value.min,
+                ('f[' ~ fieldName ~ '][v][max]'): value.max,
             }) %}
         {% else %}
             {% set path = path('adventure_index', {('f[' ~ fieldName ~ '][v]'): value}) %}
@@ -46,7 +46,7 @@
             <div class="col-4">
                 <div class="label">Publication Year</div>
                 {% if adventure.year %}
-                    {{ macros.search('year', adventure.year, null, true) }}
+                    {{ macros.search('year', {min: adventure.year, max: adventure.year}, adventure.year, true) }}
                 {% else %}
                     Unknown
                 {% endif %}
```

**Comment by cmfcmf** [comment]
And of course you don't have to use the macro if it is annoying or unfit for this use cast. You can also generate the URL directly:
```twig
<a href="{{ path('adventure_index', {
                ('f[' ~ fieldName ~ '][v][min]'): XXXX,
                ('f[' ~ fieldName ~ '][v][max]'): YYYY,
            }) }}">blah</a>
```

**Comment by damienBAUDET** [comment]
thanks for your help, I did it ! ✌️Thanks again 👍

**Comment by cmfcmf** [comment]
This was fixed in #350
